### ISSUE TITLE
CX-II CAS Not working in web mode, desktop version (windows) stuck on white screen

### ISSUE THREAD

**Issue by GamingWithAlanX** [lights0123/n-link#6]
Web console errors:
DOMException: Unable to reset the device.
(anonymous) @ 9963698.js:1
d7d2f1a.worker.js:1 wrong msgpack marker FixStr(7)
9963698.js:1 No device found
(anonymous) @ 9963698.js:1
1030099.worker.js:1 worker!
9963698.js:1 DOMException: Unable to reset the device.
(anonymous) @ 9963698.js:1
d7d2f1a.worker.js:1 wrong msgpack marker FixStr(7)
9963698.js:1 No device found
(anonymous) @ 9963698.js:1

**Comment by LeGmask** [comment]
Hmmm, lgmf! currently on windows and no issue...

**Comment by lights0123** [comment]
Yep, that's a Chrome bug in Windows—it has some issues [resetting](https://developer.mozilla.org/en-US/docs/Web/API/USBDevice/reset) USB devices on non-Linux platforms. @Adriweb submitted a different bug to Chrome after we discovered the cause of a different issue, but I don't know how to start that process for this issue.
